third_party/terraform/tests/resource_redis_instance_test.go

```diff
@@ -94,7 +91,7 @@ func TestAccRedisInstance_redisInstanceAuthEnabled(t *testing.T) {
 				ResourceName:            "google_redis_instance.cache",
 				ImportState:             true,
 				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"region"},
+				ImportStateVerifyIgnore: []string{"region", "auth_string"},
```

Comment:
Why do we ignore auth_string? Does the value change each GET? That's another case for using a datasource imo (since the volatile value won't impact the resource, and afaik the primary usecase of the auth string will be exporting it as an output)

If we don't expect the string to expire we could also use a resource to get the value and store it in state, keeping a stable value around.

Reply:
I didn't see that other issue.

I excluded the field from import because the import step was failing.

Reply:
Why was the import step failing? Do you have an error message? That's often an indicator of an actual issue.

Reply:
I'll repro it now. Also I don't like the idea of having to use a separate ds to pull the auth string. 

Also it is sent unencrypted. I know there is a new preview feature which enabled TLS connections to redis.

https://cloud.google.com/memorystore/docs/redis/auth-overview#security_and_privacy

Reply:
Also 7601 will be fixed by this PR.

Good security for Redis is achieved by using TLS + network isolation. https://redis.io/topics/security

Reply:
I figured it out. The decoder logic was broken. Pushed a fix now.